test/correctness/fuzz_simplify.cpp

```diff
@@ -184,9 +180,9 @@ Expr random_expr(FuzzedDataProvider &fdp, Type t, int depth, bool overflow_undef
                 make_absd,
             };
 
-            Expr a = random_expr(fdp, t, depth, overflow_undef);
-            Expr b = random_expr(fdp, t, depth, overflow_undef);
-            return fdp.PickValueInArray(make_bin_op)(a, b);
+            Expr a = random_expr(rng, t, depth, overflow_undef);
+            Expr b = random_expr(rng, t, depth, overflow_undef);
+            return make_bin_op[rng() % 8](a, b);
```

Comment:
Using `rng() % (sizeof(make_bin_op) / sizeof(make_bin_op[0])` here and below might be better imo, less magic constants.

Reply:
I just added a random_choice function to remove all these mods